docs/proposals/scheduling/specify-replicas/specify-replicas.md

```diff
@@ -0,0 +1,602 @@
+---
+title: Specify replicas to schedule
+
+authors:
+- "@CharlesQQ"
+
+reviewers:
+- "@robot"
+- TBD
+approvers:
+- "@robot"
+- TBD
+
+creation-date: 2024-07-08
+
+# Specify replicas to schedule
+
+## Summary
+<!--
+一种新的调度策略及其扩展API,允许调度器按照用户指定的副本数进行调度
+-->
+
+A new scheduling strategy and its extension API allow the scheduler to schedule according to the number of replicas specified by the user
+## Motivation
+<!--
+在某些场景下,用户希望服务的副本数分布是自己直接指定的结果,而不是根据权重间接计算得出的,这样更加直观
+-->
+In some scenarios, users hope that the distribution of the number of replicas in a service can be directly specified by themselves, rather than calculated indirectly based on weights, which is more intuitive.
+### Goals
+<!--
+- 扩展API,允许按照用户使用新的调度策略并指定副本数
+- 调度器支持指定副本数调度的机制
+-->
+- Extended API to allow using new scheduling strategies and specifying the number of replicas by user
+- The scheduler supports the mechanism of specifying the number of replicas for scheduling
+
+### Non-Goals
+
+## Proposal
+
+### User Stories (Optional)
+
+#### Story 1: Specify the number of replicas per cluster scenario
+<!--
+指定每个集群的副本数场景
+当deployment第一次接入Karmada的时候, 用户希望服务的副本数分布不变, 例如:
+迁移之前的服务实数分布是: `sum=7, cluster1=2,cluster2=5`
+迁移之后的服务实数分布依然是: `sum=7, cluster1=2,cluster2=5`
+-->
+
+Specify the number of replicas per cluster scenario
+When the deployment connects to Karmada for the first time, the user hopes that the distribution of the number of replicas in the service will remain unchanged, for example:
+The service real number distribution before migration is: `sum=7, cluster1=2,cluster2=5`
+The service real number distribution after migration is still: `sum=7, cluster1=2,cluster2=5`
```

Comment:
At the beginning of integrating the application with Karmada, can we use OverridePolicy to make differentiated configurations?

Reply:
> At the beginning of integrating the application with Karmada, can we use OverridePolicy to make differentiated configurations?

Use overridepolicy to modify member cluster replicas. After the next release, remove the modification of the number of replicas. The cluster topology will still change, which will cause trouble to users.